server/i18n/en.json

```diff
@@ -2818,6 +2818,14 @@
     "id": "api.saml.invalid_email_token.app_error",
     "translation": "Invalid email_token"
   },
+  {
+    "id": "api.scheduled_posts.feature_disabled",
+    "translation": "scheduled posts feature is disabled"
+  },
+  {
+    "id": "api.scheduled_posts.license_error",
+    "translation": "scheduled posts feature requires a license"
```

Comment:
q: shouldn't these texts be capitalized?

Reply:
Ah yes, done, the first letter should be capitalised. You have eagle eyes!!!